server/src/modules/rules/constants/rules.constants.ts

```diff
@@ -414,14 +455,14 @@ export class RuleConstants {
           name: 'fileAudioChannels',
           humanName: '[list] File - audio channels',
           mediaType: MediaType.MOVIE,
-          type: RuleType.NUMBER,
+          type: RuleType.NUMBER_LIST,
         },
         {
           id: 9,
           name: 'fileQuality',
           humanName: '[list] File - quality (2160, 1080,..)',
           mediaType: MediaType.MOVIE,
-          type: RuleType.NUMBER,
+          type: RuleType.NUMBER_LIST,
```

Comment:
It turns out that the getters for these just return a number so the comparison currently returns false as they aren't arrays. The API response types are set to number, but I'm sure you could have multiple audio channels. I'll have a dig into the Radarr API and see what's going on with that one. `fileQuality` will only ever be a single number as you can't have multiple files so that one needs to be set to RuleType.NUMBER.

I actually think we'll need to re-include contains for RuleType.NUMBER as it's no longer possible to compare to a list. E.g. you may want to target qualities of 720, 1080

Reply:
I can confirm that Radarr will only ever serve up a single number for audio channels, so that one can go back to RuleType.NUMBER now. I'm going to check all of the other [list]'s to see if we've got any more imposters...

Reply:
Forgot to come back to this, all of the other [list]'s are fine.

Reply:
@bitnimble fileAudioChannels is still NUMBER_LIST